data_prep/population.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# This script prepares a consolidated FGB file with total population count in
+# the most granular zone for different countries, using
+# https://github.com/Urban-Analytics-Technology-Platform/popgetter. This script
+# / process might live elsewhere at some point.
+
+set -e
+set -x
+
+# You need to build https://github.com/Urban-Analytics-Technology-Platform/popgetter-cli
+POPGETTER=/home/dabreegster/Downloads/popgetter-cli/target/release/popgetter
+
+# The metrics and coordinate systems were figured out manually. In the future,
```

Comment:
This is really helpful to hear the search process you went through, thank you!

Adding some info here for context, for each country in popgetter currently there are:
1) Original census metrics: these are identical to their original source but may have required a transformation (e.g. pivoting) so the GEO_ID was the sole "index" for rows with any other column features concatenated into single metric columns (e.g. for [NI](https://github.com/Urban-Analytics-Technology-Platform/popgetter/blob/7da293f4eb2d36480dbd137a27be623aa09449bf/python/popgetter/assets/gb_nir/__init__.py#L569))
2) Derived metrics: where the derivation is specified through a transformation of the original (e.g. for [NI](https://github.com/Urban-Analytics-Technology-Platform/popgetter/blob/7da293f4eb2d36480dbd137a27be623aa09449bf/python/popgetter/assets/gb_nir/__init__.py#L162-L210))

I think the derived metrics in case 2) need to be more easily discoverable so readily accessible for common use cases (such as "Total individuals" here). The ones transformed in 1) should also be made more easy to find using e.g. the original census table name/metric name, the variables that were in the original metric/table, etc.

I'll add the option of listing only the "derived metrics" from the CLI to the ideas issue (https://github.com/Urban-Analytics-Technology-Platform/popgetter-cli/issues/81) while enabling metrics of type 1) to be discoverable at least requires adding some data currently missing for `source_metric_id` (https://github.com/Urban-Analytics-Technology-Platform/popgetter/issues/149)